rsp/audio.s

```diff
@@ -1,1896 +0,0 @@
-.rsp
```

Comment:
Why is this file being removed? There's no point other than to ensure no custom audio ucode ever gets written. Doesn't exactly count as bloat either considering the game actually uses this file.

Reply:
`audio.s` was removed and replaced with `aspMain.o` to store the audio microcode in a way consistent with other microcodes.